cmake/modules/FindTinyXML2.cmake

```diff
@@ -5,15 +5,15 @@
 option(TINYXML2_FROM_SOURCE "Integrate TinyXML2 source code inside Fast RTPS" OFF)
 
 find_package(TinyXML2 CONFIG QUIET)
-if(TinyXML2_FOUND)
+if(TinyXML2_FOUND AND NOT THIRDPARTY)
```

Comment:
If TinyXML2 was found in the system, why not use it? is there a reason to prefer the thirdparty TinyXML2?

Reply:
The binaries in the system will be of a different architecture than the target when crosscompiling. Besides, it's easier to ship it all in one Fast-RTPS binary than figuring out the location of the system TinyXML2 binaries. So, if I want to crosscompile and later on compile Fast-RTPS natively later on, I have to uninstall and reinstall TinyXML2 every time I want to switch between native and crosscompiling.